python-frontend/typeshed_serializer/requirements.txt

```diff
@@ -6,3 +6,36 @@ toml==0.10.2
 typing-extensions==3.10.0.0
 pytest==7.2.0
 pytest-cov==4.0.0
+
+# Stub dependencies
+atomicwrites==1.4.1
+backports-abc==0.5
+certifi==2023.5.7
+characteristic==14.3.0
+click==8.1.3
+cryptography==36.0.1
+dataclasses==0.6
+enum34==1.1.10
+filelock==3.4.0
+first==2.0.2
```

Comment:
fb303 would be part of thrift, I think we would need to import it from thrift.Thrift.fb303 but maybe the easiest would be to add types-fb303 to the requirements. Not sure at all.